app/src/main/java/fr/free/nrw/commons/CommonsApplication.java

```diff
@@ -119,6 +120,8 @@ public AppLanguageLookUpTable getLanguageLookUpTable() {
     @Override
     public void onCreate() {
         super.onCreate();
+        PreferenceManager.setDefaultValues(this, R.xml.preferences, false);
```

Comment:
Please inject and use `JSONKvStore` instead of using shared prefs directly. We have added a tidy wrapper around the shared prefs. See our wiki for ref: https://github.com/commons-app/apps-android-commons/wiki/Code-style#shared-preferences

For eg:

```
@Inject @Named("default_preferences") JsonKvStore defaultKvStore;
```

Reply:
I removed this as I don't know how to set default values from xml to `defaultKvStore` in CommonsAplication.java